image-sync/oc-mirror/test/ocp-image-set-config.yml

```diff
@@ -29,3 +29,7 @@ mirror:
   - name: registry.redhat.io/redhat/certified-operator-index:v4.18
```

Comment:
As another side comment, I see there's another `channels` section. Is that being leveraged? does that conflict with the ACR caching that we have for OCP images themselves? I am trying to make sure that executing whatever needs to be executed doesn't cause any conflict and/or that we clean up elements that do not apply anymore to not cause confusion.